docs/technical_documents/hybrid_a_star_en.md

```diff
@@ -0,0 +1,105 @@
+Generate coarse trajectory on the open space 
+
+# Introduction
+The goal of this part is to generate the coarse trajectory in the open space. This part of the code consists of four parts: node3d，grid_search，reeds_shepp_path，hybrid_a_star. Hybrid_a_star the most important part of generating the coarse trajectory, it calls the grid_search and reeds_shepp_path.
```

Comment:
maybe: node3d，grid_search，reeds_shepp_path and hybrid_a_star.
Hybrid_a_star is the most important